code/src/App.js

```diff
@@ -1,9 +1,28 @@
-import React from 'react'
+import React, { useState } from 'react'
+import musicData from './data.json'
+import { Header } from './components/Header'
+import { Album } from './components/Album'
+import { EmptyState } from './components/EmptyState'
```

Comment:
There is a convention which says that we should divide outer dependencies (those located in `node_modules`) from local dependencies (all local files that we created - components, functions, data such as arrays) by one empty line. Moreover, `style.css` should be separated as well and located below local dependencies. Ideally, it should look like this :

```
import React from 'react';
import ReactDOM from 'react-dom';

import MyComponent1 from './components/MyComponent1';
import MyComponent2 from './components/MyComponent2';
import MyComponent3 from './components/MyComponent3';

import './style.css';
```

So, final order is :

Outer dependencies
Local dependencies
Style file(s)